Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build
+FROM rust:1.73 AS build
+
+WORKDIR /grafbase
+
+COPY ./cli ./cli
+COPY ./engine ./engine
+
+COPY ./cli/Cargo.lock ./cli/Cargo.lock
+COPY ./cli/Cargo.toml ./cli/Cargo.toml
+COPY ./engine/Cargo.lock ./engine/Cargo.lock
+COPY ./engine/Cargo.toml ./engine/Cargo.toml
+
+WORKDIR /grafbase/cli
+
+RUN cargo build --release
+
+# Run
+FROM debian:bookworm AS run
```

Comment:
try using one of the following and see how it goes:

- distroless images
- alpine images
- if you're having fun, try scratch images

should definitely decrease the size

Reply:
We're not statically linking (unless I've missed something) so probably limits to how far away from debian we can go before we start running into problems